app/assets/styles/global/_content.scss

```diff
@@ -0,0 +1,204 @@
+/* ==========================================================================
+   Homepage: Content
+   ========================================================================== */
+
+.page--homepage {
+  .inpage__header {
+    background-image: url(../graphics/content/page-hero-default.png);
+  }
+}
+
+.showcase {
+  @include media(medium-up) {
+    @include col(9/12);
+    @include shift(1.5/12);
+  }
+
+  @include media(large-up) {
+    @include col(8/12);
+    @include shift(2/12);
+  }
+}
+
+
+/* Exhibit
+   ========================================================================== */
+
+.exhibit {
+  @extend .clearfix;
+  box-shadow: 0 0 0 2px $base-alpha-color;
+  border-radius: $base-border-radius;
+  background: #fff;
+  font-size: 1rem;
+  line-height: 1.5;
+  margin: 0 2/12*100% 1rem 2/12*100%;
+  @include media(medium-down) {
+    margin: 0 1/12*100% 1rem 1/12*100%;
+  }
+
+  li > & {
+    margin: 0;
+  }
+
+  &--clipped {
+    overflow: hidden;
+  }
+}
+
+.exhibit__content,
+.exhibit__caption,
+.prose .exhibit__caption {
+  padding: $global-spacing;
+  clear: both;
+
+  > *:first-child {
+    margin-top: 0;
+  }
+
+  > *:last-child {
+    margin-bottom: 0;
+  }
+
+  @include media(medium-up) {
+    padding: $global-spacing * 1.5;
+  }
+
+  a.blue {
```

Comment:
You should avoid using classes based on styles. The element should be styled in relation to its context or with a more generic class that can scale.